src/plone/restapi/services/types/get.py

```diff
@@ -72,6 +72,7 @@ def __call__(self, expand=False):
         result["types"] = [
             {
                 "@id": f"{portal_url}/@types/{fti.getId()}",
+                "@type": fti.getId(),
```

Comment:
This seems misleading. If JSON data is labeled with `@type: "X"` I would expect it to include the fields from the X type. But this includes the properties of an FTI object. The type is FTI, not a specific portal_type.

Reply:
@davisagli The idea is to avoid getting `@type` from `@id` like https://github.com/plone/volto/blob/52975f103cb7520fa375bbd5ae2a844b6e446ed4/src/components/manage/Toolbar/Types.jsx#L24  See: https://github.com/plone/plone.restapi/issues/1322

Reply:
@nileshgulia1 My point is that `@type` has a meaning in JSON-LD: it identifies the format of the object that is represented in the JSON there. But you are using it for something else (the id of the portal type that is defined by this FTI) ... uh, it's a bit hard to explain because there are two different kinds of types we're talking about here

Can you change it to use `id` instead? Then I am fine with it.
```suggestion
                "id": fti.getId(),
```